performance_tests/run_tests.py

```
@@ -0,0 +1,254 @@
+
+# Add parent directory to path
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+def run_command(cmd: List[str], description: str) -> Dict[str, Any]:
+    """Run a command and return results."""
+    print(f"\n🚀 {description}")
+    print(f"Command: {' '.join(cmd)}")
+    
+    start_time = time.time()
+    try:
+        result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)